build-support/bin/native/bootstrap.sh

```diff
@@ -62,6 +62,11 @@ function ensure_build_prerequisites() {
     sh ${rustup} -y --no-modify-path 1>&2
     rm -f ${rustup}
     ${RUSTUP_HOME}/bin/rustup override set stable 1>&2
+
```

Comment:
I may be missing something, but for portability (distribution), changes should be here: https://github.com/pantsbuild/pants/blob/master/build-support/bin/native/generate-bintray-manifest.sh#L81-L82